src/android/toga_android/widgets/canvas.py

```diff
@@ -0,0 +1,147 @@
+from ..libs import activity
+from ..libs.android.graphics import Paint, Paint__Style, Path
+from .base import Widget
+
+# Arbitrary scale factor; to be made more specific in the future.
+SCALE_FACTOR = 5
```

Comment:
This stood out for me; I'm guessing this is a factor that should be tied to a combination of screen DPI and size? As it stands, it looks like the Android canvas draws "bigger" than the iOS equivalent

Reply:
Definitely. I hope to fix this is in a follow-up.